CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+## Contributing
+
+[fork]: https://github.com/education/classroom/fork
+[pr]: https://github.com/education/classroom/compare
+[style]: https://github.com/bbatsov/ruby-style-guide
+
+Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+
+## Submitting a pull request
+
+0. [Fork][fork] and clone the repository
+0. Configure and install the dependencies: (See the [README](README.md) for more details)
+0. Make sure the tests pass on your machine: `script/test`
+0. Make sure the linters pass on your machine: `script/linter`
+0. Create a new branch: `git checkout -b my-branch-name`
+0. Make your change, add tests, and make sure the tests still pass
+0. Push to your fork and [submit a pull request][pr]
+0. Pat your self on the back and wait for your pull request to be reviewed and merged.
+
+Here are a few things you can do that will increase the likelihood of your pull request being accepted:
+
+- Follow the [style guide][style].
+- Write tests.
+- Keep your change as focused as possible. If there are multiple changes you would like to make that are not dependent upon each other, consider submitting them as separate pull requests.
+- Write a [good commit message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+
```

Comment:
Could use a section on reporting issues, what makes a good report, and searching for duplicates.